pkg/chartvalues/apiextensions_app_e2e.go

```diff
@@ -0,0 +1,84 @@
+package chartvalues
+
+import (
+	"github.com/giantswarm/e2etemplates/internal/render"
+	"github.com/giantswarm/microerror"
+)
+
+type APIExtensionsAppE2EConfig struct {
+	App         APIExtensionsAppE2EConfigApp
+	AppCatalog  APIExtensionsAppE2EConfigAppCatalog
+	AppOperator APIExtensionsAppE2EConfigAppOperator
+	ConfigMap   APIExtensionsAppE2EConfigConfigMap
+	Namespace   string
+	Secret      APIExtensionsAppE2EConfigSecret
+}
+
+type APIExtensionsAppE2EConfigApp struct {
+	Config    APIExtensionsAppE2EConfigAppConfig
+	Name      string
+	Namespace string
```

Comment:
I think the Version should also be configurable. We'll need that in the e2e tests to test updating apps.

Reply:
@rossf7 Which version do you mean? In label?
```
      labels:
        app: "prometheus"
        app-operator.giantswarm.io/version: "1.0.0"
```

Reply:
Ah, in spec. 
Sorry
```
   spec:
      catalog: "giantswarm"
      name: “prometheus”
      namespace: “monitoring”
      version: "3.2.1"
```

Reply:
No worries but yes we need the version to generate the tarball URLs. So we need to test that.